remoteappmanager/jupyterhub/spawners.py

```diff
@@ -32,24 +31,29 @@ def cmd(self):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-
         # We get the first one. Strangely enough, jupyterhub has a table
         # containing potentially multiple proxies, but it's enforced to
         # contain only one.
-        self.proxy = self.db.query(orm.Proxy).first()
+        self.proxy = self.db.query(ConfigurableHTTPProxy).first()
```

Comment:
This won't work - we need to somehow get a reference to the application's `JupyterHub.proxy` attribute

Reply:
Actually we only really need a reference to the Proxy API token and url